src/main/java/org/broadinstitute/hellbender/utils/test/BaseTest.java

```diff
@@ -194,6 +194,13 @@ public String getToolTestDataDir(){
     }
 
     /**
+     * Returns the location of the resource directory. The default implementation points to the common directory for tools.
+     */
+    public static File getTestDataDir(){
+        return new File("src/test/resources/org/broadinstitute/hellbender/tools/");
+    }
```

Comment:
Since this is `BaseTest`, not `CommandLineProgramTest`, it's not appropriate to have the default test data directory be the tools directory (since many subclasses do not test actual tools). `BaseTest.getTestDataDir()` should return the root of the test data dir (ie., `src/test/resources`), and `CommandLineProgramTest` or other subclasses that test tools should override to point to the tools dir.

Also, you should be using the pre-existing constant `BaseTest.publicTestDir` here.